.gitlab-ci.yml

```diff
@@ -349,6 +385,7 @@ test_python_linux_python27:
     - build_wheel_linux_python27
   script:
     - apk add bash
+    - export TC_ENABLE_S3_TESTS=1
```

Comment:
I don't think we want `TC_ENABLE_S3_TESTS=1` in master; last I checked this will cause test failures in public CI.

Reply:
Good catch. This was the result of a bad merge. I've removed all three occurrences.